### ISSUE TITLE
Contact Us link

### ISSUE THREAD

**Issue by val99erie** [curationexperts/alexandria-v2#16]
There is a "Contact Us" link in the footer that is currently broken.  We need to either create the form for it, or remove the link from the page.

**Comment by acozine** [comment]
Recreate form from current site:
http://alexandria.ucsb.edu/ucsb/ucsb_form1

**Comment by jcoyne** [comment]
Code here: https://stash.library.ucsb.edu/projects/DR/repos/adrl/browse/app/controllers/ucsb_controller.rb

**Comment by val99erie** [comment]
@acozine & @mark-dce , here are some notes that you might want to know for the demo:

* The "Contact Us" form on Alexandria V1 contains an invisible "zipcode" field, which is meant for spam detection.  Humans won't fill in the field because it's not displayed on the form, but bots might.  Suspected spam messages will get a special flag on the subject line to allow ADRL staff to filter spam away from real inquiries.  When I wrote the "Contact Us" page for V2, I wrote it the same way, so it should behave exactly as V1 did.

* Emails that come from the "Contact Us" form have a special "[ADRL]" marker in the subject line.  I wrote V2 to behave just like V1, but with one exception:  In V2, if the Rails environment is not production, it will have "[ADRL Demo]" instead.  I wanted us to be able to test email on our demo environment without confusing anyone.
